test/SharedFx.UnitTests/SharedFxTests.cs

```diff
@@ -1,14 +1,76 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Reflection;
 using Newtonsoft.Json.Linq;
 using Xunit;
 
 namespace Microsoft.AspNetCore
 {
     public class SharedFxTests
     {
+
+        [Theory]
+        [MemberData(nameof(GetSharedFxConfig))]
+        public void BaselineTest(SharedFxConfig config)
+        {
+            var previousVersion = TestData.GetPreviousAspNetCoreReleaseVersion();
+            var url = $"https://dotnetcli.blob.core.windows.net/dotnet/aspnetcore/Runtime/" + previousVersion + "/aspnetcore-runtime-internal-" + previousVersion + "-win-x64.zip";
+            var zipName = "assemblies.zip";
+            var nugetAssemblyVersions = new Dictionary<string, Version>();
+
+            var root = TestData.GetDotNetRoot();
+            var dir = Path.Combine(root, "shared", config.Name, config.Version);
+
+            using (var testClient = new WebClient())
+            {
+                testClient.DownloadFile(url, zipName);
+            }
+
+            var zipPath = Path.Combine(AppContext.BaseDirectory, zipName);
+
+            if (!Directory.Exists("unzipped"))
```

Comment:
Should this create a new, random folder somewhere? What if there's already junk there from a previous failed run?

Reply:
Also, where is the "current directory" when this is called? Looks like "current directory" and "BaseDirectory" are used but are they necessarily the same?